app/models/activity.py

```diff
@@ -1,31 +1,50 @@
 from .. import db
 from sqlalchemy import Column, Integer, Boolean, DateTime
-from .user import User
-from .room_request import *
 from datetime import datetime
+from .user import User
 
 
-class  Activity(db.Model):
-    _tablename_ = "activty"
+class Activity(db.Model):
+    _tablename_ = "activity"
 
     id = db.Column(db.Integer, primary_key=True)
-    created_date = db.Column(DateTime, default= datetime.utcnow)    
+    created_at = db.Column(DateTime, default=datetime.utcnow)
     text = db.Column(db.String())
-    commenter = db.Column(db.String())
```

Comment:
I removed this because it's somewhat redundant data - we can instead do `activity.user.first_name` and `activity.user.last_name`. That way, if the user changes their name, the activity updates as well.